src/app/models/__tests__/form.server.model.spec.ts

```diff
@@ -1676,5 +1676,45 @@ describe('Form Model', () => {
         expect(updatedForm).toBeNull()
       })
     })
+
+    describe('updateFormCollaborators', () => {
+      it('should return the form with an updated list of collaborators', async () => {
+        // Arrange
+        const newCollaborators = [
+          {
+            email: `fakeuser@${MOCK_ADMIN_DOMAIN}`,
+            write: false,
+          },
+        ]
+
+        // Act
+        const actual = await validForm.updateFormCollaborators(newCollaborators)
+
+        // Assert
+        const actualPermissionsWithoutId = map(
+          actual.permissionList,
+          (collaborator) => pick(collaborator, ['email', 'write']),
+        )
+        expect(actualPermissionsWithoutId).toEqual(newCollaborators)
+      })
+
+      it('should return an error if validation fails', async () => {
+        // Arrange
+        const newCollaborators = [
+          {
+            email: `fakeuser@fakeemail.com`,
+            write: false,
+          },
+        ]
+
+        // Act
+        const actual = await validForm
+          .updateFormCollaborators(newCollaborators)
+          .catch((error) => error)
+
+        // Assert
+        expect(actual).toBeInstanceOf(mongoose.Error.ValidationError)
```

Comment:
a more canonical way to write this would be to use jest's `.rejects`

Reply:
good point, will do!